notebooks/build/covid-19-el-paso.ipynb

```diff
@@ -0,0 +1,6756 @@
+{
```

Comment:
<p>I mentioned this elsewhere, but I think I'd like to fuzz the model projections rather than fuzzing how we use them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>